pytest_tests/tests/services/s3_gate/test_s3_bucket.py

```diff
@@ -27,15 +27,23 @@ class TestS3GateBucket(TestNeofsS3GateBase):
     def test_s3_create_bucket_with_ACL(self):
         with allure.step("Create bucket with ACL = private"):
             acl = "private"
-            bucket = s3_gate_bucket.create_bucket_s3(self.s3_client, object_lock_enabled_for_bucket=True, acl=acl, bucket_configuration="rep-1")
-            bucket_acl = s3_gate_bucket.get_bucket_acl(self.s3_client, bucket)
-            assert_bucket_s3_acl(
-                acl_grants=bucket_acl, permitted_users="CanonicalUser", acl=acl
+            bucket = s3_gate_bucket.create_bucket_s3(
+                self.s3_client,
+                object_lock_enabled_for_bucket=True,
+                acl=acl,
+                bucket_configuration="rep-1",
```

Comment:
Whitespace. Is there any `go fmt` or `dotnet format` for Python? Can we run it once for our codebase, store a config for it and _never_ have these hunks again?

Reply:
ef04906beaba71ca8d6e633b01601456f0582f47 tried to do that, but looks like it wasn't sufficient.

Reply:
We have _black_ formatter, and I always use it, that's why I always have some not related to PR changes. For this particular PR I ran it for all files.

Reply:
The way to do it:
 * run it once for _everything_
 * commit these whitespace changes only
 * go on from this base

The problem is mixing whitespace with real changes, it's a mess to review.